.env.example

```diff
@@ -0,0 +1,32 @@
+# Local Auth serv
+REEARTH_AUTH0_DOMAIN=http://localhost:8080
+REEARTH_AUTH0_AUDIENCE=http://localhost:3000
+REEARTH_AUTH0_CLIENTID=reearth-authsrv-client-default
+REEARTH_AUTH0_CLIENTSECRET=
+REEARTH_AUTH0_WEBCLIENTID=
+
+# config for auth client
+REEARTH_AUTH=[{"ISS":"http://localhost:8080","AUD":["http://localhost:8080"]}]
+
+# config for auth server
+CAOS_OIDC_DEV=true
+REEARTH_AUTHSRV_DISABLED=false
+REEARTH_AUTHSRV_UIDOMAIN=http://localhost:3000
+REEARTH_AUTHSRV_DOMAIN=http://localhost:8080
+REEARTH_AUTHSRV_KEY=test
+
+#REEARTH_MAILER available options: [log, smtp, sendgrid]
+REEARTH_MAILER=log
+REEARTH_SMTP_EMAIL=noreplay@test.com
+
+#SendGrid config
+#REEARTH_SMTP_HOST=smtp.sendgrid.net
+#REEARTH_SMTP_PORT=587
+#REEARTH_SMTP_SMTPUSERNAME=apikey
+#REEARTH_SMTP_PASSWORD=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
```

Comment:
```suggestion
#REEARTH_MAILER=smtp

# SendGrid config
#REEARTH_SMTP_EMAIL=noreplay@test.com
#REEARTH_SMTP_HOST=smtp.sendgrid.net
#REEARTH_SMTP_PORT=587
#REEARTH_SMTP_SMTPUSERNAME=apikey
#REEARTH_SMTP_PASSWORD=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
```

REEARTH_SMTP_EMAIL is not used when default logger mailer is used

BTW some password is described here. Is it safe?